scripts/update-docker-images.sh

```diff
@@ -1,49 +1,34 @@
 #!/usr/bin/env bash
 set -euv
 
+if [[ ${TRAVIS_TAG} != v* ]]; then
+  echo "Not on a git tag"
+  exit 1
+fi
+
 SCALA212_VERSION="$(grep -oP '(?<=def scala212 = ")[^"]*(?<!")' project/Settings.scala)"
 SCALA211_VERSION="$(grep -oP '(?<=def scala211 = ")[^"]*(?<!")' project/Settings.scala)"
 
-VERSION="$(git describe --tags --abbrev=0 --match 'v*' | sed 's/^v//')"
-
-mkdir -p target
-cd target
-
-git clone "https://$GH_TOKEN@github.com/almond-sh/docker-images.git" -b template
-cd docker-images
-
-BRANCHES=()
-
-for sv in "$SCALA211_VERSION" "$SCALA212_VERSION"; do
-
-  BRANCH="almond-$VERSION-scala-$sv"
-  git checkout -b "$BRANCH" template
-  BRANCHES+=("$BRANCH")
+ALMOND_VERSION="$(git describe --tags --abbrev=0 --match 'v*' | sed 's/^v//')"
 
-  mv Dockerfile Dockerfile.tmp
-  cat Dockerfile.tmp |
-    sed 's@{SCALA_VERSION}@'"$sv"'@g' |
-    sed 's@{VERSION}@'"$VERSION"'@g' > Dockerfile
-  rm -f Dockerfile.tmp
+DOCKER_REPO=almondsh/almond
+IMAGE_NAME=${DOCKER_REPO}:${ALMOND_VERSION}
 
-  git add Dockerfile
-  git commit -m "almond $VERSION, scala $sv"
-done
+sbt '+ publishLocal'
```

Comment:
Do we want to do that for releases? They should already be on Maven Central when this stage runs.

Maybe we could run that stage for snapshots (and push them with a `snapshot` tag say), and rely on `publishLocal` only for them.

Reply:
Right now it runs in parallel to the upload-launcher (see my comment about the stage) so the artifacts aren't available yet. I can change it back to run afterwards and use the published artifacts. Then only do a `publishLocal` for snapshots builds as you've suggested.

Reply:
I guess we can run them sequentially. Doing so allows to avoid the `+publishLocal`, so maybe it's not even slower…

Reply:
The script in combination with .travis.yml differentiates now between releases and a push to master. `+publishLocal` runs only on master now and creates a snapshot docker image from the current HEAD. On a tag, it should grab the published artifacts instead.